doc/Makefile

```diff
@@ -5,7 +5,7 @@
 SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
 PAPER         =
-BUILDDIR      = _build
```

Comment:
Why did you change this? I will break the Github scripts to automatically build the doc.